tensorflow/lite/micro/kernels/Makefile.inc

```diff
@@ -102,6 +101,7 @@ tensorflow/lite/micro/kernels/svdf_test.cc \
 tensorflow/lite/micro/kernels/tanh_test.cc \
 tensorflow/lite/micro/kernels/transpose_test.cc \
 tensorflow/lite/micro/kernels/unpack_test.cc \
+tensorflow/lite/micro/kernels/while_test.cc \
```

Comment:
Merge error:? broadcast_to_test.cc is removed

Reply:
Fixed.